src/app/dataset-view/additional-components/data-component/data-component.ts

```diff
@@ -72,9 +75,14 @@ export class DataComponent extends BaseComponent implements OnInit {
         this.buildSqlRequestCode();
     }
 
-    onInitEditor(editor: monaco.editor.IStandaloneCodeEditor): void {
+    public changeLimit(limit: number): void {
+        this.defaultLimit = limit;
```

Comment:
If that became a variable, then it needs a new name. "Default" is something that does not get changed.

Reply:
Not actual